### PR TITLE
`berkeley`: Unskip `test_submit_workflow_activities` test and make it work with Berkeley schema

### PR DESCRIPTION
**Pull request by aclum**
# Description

This PR fixes test_submit_workflow_activities to be compatible with berkeley schema

Fixes # [(issue)](https://github.com/microbiomedata/nmdc-runtime/issues/683)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration, if it is not simply `make up-test && make test-run`.

- [ ] Test A
- [ ] Test B

testing is from github acations

**Configuration Details**: none

# Definition of Done (DoD) Checklist:
A simple checklist of necessary activities that add verifiable/demonstrable value to the product by asserting the quality of a feature, not the functionality of that feature; the latter should be stated as acceptance criteria in the issue this PR closes. Not all activities in the PR template will be applicable to each feature since the definition of done is intended to be a comprehensive checklist. Consciously decide the applicability of value-added activities on a feature-by-feature basis.

- [ ] My code follows the style guidelines of this project (have you run `black nmdc_runtime/`?)
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (in `docs/` and in <https://github.com/microbiomedata/NMDC_documentation/>?)
- [ ] I have added tests that prove my fix is effective or that my feature works, incl. considering downstream usage (e.g. <https://github.com/microbiomedata/notebook_hackathons>) if applicable.
- [x] New and existing unit and functional tests pass locally with my changes (`make up-test && make test-run`)

### CODE REVIEW

**Comment by aclum** [comment]
@PeopleMakeCulture  intersted in your feedback on this one. Im not sure why there would be a race condition, I've updated it for berkeley and the test passes now. Let me know if its okay to leave the pytest.skip commented out.

File: tests/test_api/test_endpoints.py [hunk 1]
```diff
@@ -270,12 +270,12 @@ def test_submit_changesheet():
     assert True
 
 
-@pytest.mark.skip(
-    reason="Skipping because race condition causes  http://fastapi:8000/nmdcschema/ids/nmdc:wfrqc-11-t0tvnp52.2 to 404?"
-)
+#@pytest.mark.skip(
+#    reason="Skipping because race condition causes  http://fastapi:8000/nmdcschema/ids/nmdc:wfrqc-11-t0tvnp52.2 to 404?"
+#)
```
Comment (dwinston) [inline tests/test_api/test_endpoints.py:275]: Delete these lines. They are recoverable via git history.

**Review by dwinston — APPROVED** [review]
Seems fine.

File: tests/test_api/test_endpoints.py (hunk 1 above)
Comment (eecavanna) [reply tests/test_api/test_endpoints.py:275]: Thanks, @dwinston. I'll do that here now @aclum (via `github.dev`)

Comment (eecavanna) [inline tests/test_api/test_endpoints.py]: On second thought, I'll do it via a "suggestion" here in the PR.

```suggestion
```

Comment (eecavanna) [reply tests/test_api/test_endpoints.py]: On third thought, I'll, uh... go get some water.

**Pull request merged**